build/buildRunTimeImages.sh

```diff
@@ -105,7 +116,7 @@ for dockerFile in $dockerFiles; do
         acrRuntimeImageTagNameRepo="$ACR_PUBLIC_PREFIX/$getTagName_result"
 
         # Tag the image to follow a similar format to .../python:3.7-Oryx-CI.20191028.1
-        docker tag "$localImageTagName" "$acrRuntimeImageTagNameRepo-$uniqueTag"
+        docker tag "$localImageTagName" "$acrRuntimeImageTagNameRepo-$uniqueTag-$runtimeImageType"
```

Comment:
Should you also have unique artifacts file generated? i.e `ACR_RUNTIME_IMAGES_ARTIFACTS_FILE`...if this file is executed in 2 separate jobs then there would be race condition